core/src/main/scala/cell/CellCompleter.scala

```diff
@@ -14,6 +14,8 @@ trait CellCompleter[K <: Key[V], V] {
    */
   def cell: Cell[K, V]
 
+  def init: (Cell[K, V]) => Outcome[V]
```

Comment:
Why should one be able to get the initializer closure by calling a public method? The API surface should really be minimized. Thus: could `def init` be `private[cell]`?